target/bin/delmailuser

```diff
@@ -144,17 +144,18 @@ use 'sudo docker exec mailserver rm -R /var/mail/${DOMAIN}/${USER}'"
   then
     if rm -R "/var/mail/${DOMAIN}/${USER}"
     then
-      echo "Mailbox deleted."
+      _log 'info' 'Mailbox deleted'
     else
-      echo "Mailbox couldn't be deleted." >&2
+      _log 'error' 'Mailbox could not be deleted' >&2
       ERROR=true
     fi
-    rmdir "/var/mail/${DOMAIN}" 2>/dev/null || true
+    rm -rf "/var/mail/${DOMAIN}"
```

Comment:
Right, this would not be ideal. I will revert it back to `rmdir`.

The purpose of `|| true` is to mask the return value. `rmdir` will fail and exit with `1` if the directory was not empty, and therefore, the whole command would fail. This is however not needed here since we do not use `set -e`.

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7